test/infrastructure/app_feedback/app_feedback_repository_test.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:io';
+
+import 'package:agora/domain/feedback/device_informations.dart';
+import 'package:agora/domain/feedback/feedback.dart';
+import 'package:agora/infrastructure/app_feedback/repository/app_feedback_repository.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../utils/dio_utils.dart';
+
+void main() {
+  final dioAdapter = DioUtils.dioAdapter();
+  final httpClient = DioUtils.agoraDioHttpClient();
+  final bug = AppFeedback(
+    type: AppFeedbackType.bug,
+    description: 'description',
+  );
+  final deviceInfo = DeviceInformations(appVersion: 'appVersion', model: 'model', osVersion: 'osVersion');
+
+  test('when success, should return true', () async {
```

Comment:
Là par contre je ferai bien un cas `when type is not bug, should send null deviceInfo`